requirements.txt

```diff
@@ -166,14 +167,14 @@ rich==12.4.4
     # via funding-service-design-utils
 ruamel-yaml==0.17.21
     # via prance
-ruamel-yaml-clib==0.2.6
-    # via ruamel-yaml
 s3transfer==0.6.1
     # via boto3
 semver==2.13.0
     # via prance
-sentry-sdk[flask]==1.9.9
-    # via funding-service-design-utils
+sentry-sdk[flask]==1.31.0
```

Comment:
As discussed with Sarah - this needs removing from notifications, utils version needs a release first

Reply:
removed

Reply:
This is right - it will appear in `requirements.txt` as it's brought in from utils - see the comment on the line below